bindings/java/src/org/sleuthkit/datamodel/blackboardutils/FileAttachment.java

```diff
@@ -74,10 +74,15 @@ public FileAttachment(SleuthkitCase caseDb, Content dataSource, String pathName)
 
 		//normalize the slashes.
 		this.filePathName = normalizePath(pathName);
-
+		
+		
 		String fileName = filePathName.substring(filePathName.lastIndexOf('/') + 1);
-		String parentPathSubString = filePathName.substring(0, filePathName.lastIndexOf('/'));
+		if (fileName.isEmpty()) {
```

Comment:
I'm guessing this came up. How do we get an attachment path with no file name component?

Reply:
Ah, I see the answer in Jira.